exercises/strings/rot13/rot13.js

```diff
@@ -31,9 +31,16 @@
 function rot13(string) {
   // This is your job. :)
 }
+let sourceAlphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+let targetAlphabet = 'NOPQRSTUVWXYZABCDEFGHIJKLM';
+let output = '';
+for (input = sourceAlphabet){
+  let output = output + targetAlphabet.Indexof()
```

Comment:
Style note: this line can look cleaner with `let output += targetAlphabet.Indexof()`

Reply:
Make sure look read https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf, also